manuscript.md

```diff
@@ -111,7 +113,7 @@ It could be misunderstood that the software is open-source, when in reality, all
 
 We recommend for the proteomics community as the gold standard for proteomics software development the following: 
 
-- Use a standard OSS licenses like the Apache 2.0, MIT, GPL, LGPL, etc; a list of them can be found in (https://opensource.org/licenses). 
+- Use a standard OSS licenses like the Apache 2.0, MIT, BSD, LGPL, GPL, etc; a list of them can be found in (https://opensource.org/licenses). 
```

Comment:
Not a huge fan of GPL in research software... but maybe not worth going into detail here.

Reply:
Agreed.